src/controllers/discussion-controller.js

```diff
@@ -0,0 +1,168 @@
+const Discussions = require('../models/discussions');
+const Accounts = require('../models/accounts');
+const {StatusCodes} = require ('http-status-codes');
+const mongoose = require('mongoose');
+const {responseServer, responseNotFound, responseGeneral} = require('../helpers/response-result');
+
+const validateExistsDiscussion = (request, response, next) => {
+    const {idDiscussion} = request.body;
+    Discussions.findById(idDiscussion, {_id:1})
+    .then(function (discussion) {  
+        if(discussion){
+            return next();
+        }
+        return responseGeneral(response, StatusCodes.BAD_REQUEST, "La discusion no existe");
+    })
+    .catch(function (error){
+        return responseServer(response, error);
+    });
+}
+
+const getDiscussionsFilters = async (request, response) => {
+    const filter = request.params.filter;
+    let queryDiscussions = {numberComments:{ $gte: 15 }};
+    if(filter == "news"){
+        const dateNow = new Date();
+        const dateCreation = new Date(dateNow.getTime() - (dateNow.getTimezoneOffset() * 60000 )).toISOString().slice(0, 10);
+        queryDiscussions = {dateCreation:dateCreation};
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        queryDiscussions = {dateCreation};
```